proto/caraml/upi/v1/upi.proto

```diff
@@ -132,3 +108,14 @@ message ModelMetadata {
   // Model version that produce prediction
   string version = 2;
 } 
+
+// Transformer input contains additional information that can be used to enrich prediction_table using standard transformer.
+// All tables and variables within transformer input will be imported to the standard transformer runtime automatically.
+message TransformerInput {
```

Comment:
should it be `TransformerInputs`?

Reply:
It's debatable, I think message name should be singular irrespective of whether it contains repeated fields, but I couldn't find this in protobuf style guide to back it up.